src/stores/SetupEncryptionStore.ts

```diff
@@ -151,6 +155,8 @@ export class SetupEncryptionStore extends EventEmitter {
                         // to advance before this.
                         await cli.restoreKeyBackupWithSecretStorage(backupInfo);
                     }
+
+                    await initializeDehydration();
```

Comment:
Yes, currently it's just for newly-logged-in users, because we need to access Secret Storage, and we don't cache the Secret Storage key, and we don't want to surprise users with a popup asking for the key.